Exercise 2/Start/BackgroundDownload/Info.plist

```diff
@@ -1,59 +1,61 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd"[]>
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
```

Comment:
I wonder where this `[]` came from. I'm guessing that's a bug. It's still present in the Completed project, so I'll remove it there as well.

Reply:
Uh, yes. Thanks. But it did work, right? Strange.